File: alarmClock/AlarmClock.java
package alarmClock;

public class AlarmClock {
    //A variável "day" representará os dias da semana, de modo que domingo = 0, segunda = 1, terça = 2..., sabendo disso,
    //retorne uma String que seguirá a seguinte lógica: se for dia de semana retorne "7:00", caso não "10:00", porém se
    //a variável "vacation" for true, nos dias de semana deve-se retornar "10:00", caso não "off".

    public String alarmClock(int day, boolean vacation) {
        if (!vacation && day!=0 && day!=6) return "7:00";
        else if (vacation && (day==0 || day==6)) return "off";
        return "10:00";
    }

}

File: Python/Logic-1/alarm_clock.py
"""
Dada um número inteiro "day" que segue a lógica: 0-> domingo, 1-> segunda..
6-> sábado. E um tipo booleano "vacation", Retorne os seguintes valores com
suas lógicas correspondentes:
"7:00"-> quando for um dia da semana e "vacation" for false
"10:00"-> quando for um fim de semana e "vacation" false ou um dia de semana e
"vacation" true
"off"-> quando for um fim de semana e "vacation" for true
"""


def alarm_clock(day, vacation):
    semana = [n for n in range(1, 6)]
    if vacation:
        if day in semana:
            return "10:00"
        return "off"
    if day in semana:
        return "7:00"
    return "10:00"


print(alarm_clock(0, True))